Example/JustLog.xcodeproj/project.pbxproj

```diff
@@ -450,7 +450,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.4;
```

Comment:
I just realised, you'll also need to update the minimum versions in `Package.swift`

Reply:
Yeah, you are right! Updated it!